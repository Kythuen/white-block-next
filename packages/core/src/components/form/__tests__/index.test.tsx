import { describe, expect, it } from 'vitest'
import { mount } from '@vue/test-utils'
import Form from '../form.vue'
// import { useForm } from '../use'

describe('# Form', () => {
  describe('## Base', () => {
    it('### default config', () => {
      const wrapper = mount({
        render() {
          return <Form />
        }
      })
      // expect(wrapper.classes()).toEqual(['wb-form'])
      // expect(wrapper.props()).toEqual({})
      expect(wrapper.element).toMatchSnapshot()
    })
  })

  // describe('## Slots', () => {})

  // describe('## Events', () => {})

  // describe('## Props', () => {})

  // describe('## Composables', () => {})
})
