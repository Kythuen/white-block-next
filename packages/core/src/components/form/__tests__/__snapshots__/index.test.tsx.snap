// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`# Form > ## Base > ### default config 1`] = `
<form
  class="wb-form wb-form--normal"
  flex="~ col"
>
  
  <!---->
  
</form>
`;
